src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -198,6 +200,49 @@ def endAuction():
 
 This example should give you a sense of what Vyper contract syntax is like. For a more detailed description of the functions and variables, [see the docs](https://vyper.readthedocs.io/en/latest/vyper-by-example.html#simple-open-auction).
 
+## Yul and Yul+ {#yul}
+
+**Yul**
+- Intermediate language for Ethereum.
+- Supports EVM and eWASM, and it is designed to be a usable common denominator of both platforms.
+- Good target for high-level optimisation stages that can benefit all target platforms equally.
+
+**Yul+**
+- A low-level, highly efficient extension to Yul.
+- Initially designed for an optimistic rollup contract.
```

Comment:
Can we link to layer 2 scaling page here: `/en/developers/docs/layer-2-scaling/`